src/main/java/de/retest/recheck/RecheckProperties.java

```diff
@@ -0,0 +1,84 @@
+package de.retest.recheck;
+
+import java.util.List;
+
+import org.aeonbits.owner.Config.Sources;
+import org.aeonbits.owner.ConfigCache;
+import org.aeonbits.owner.ConfigFactory;
+import org.aeonbits.owner.Mutable;
+
+import de.retest.recheck.configuration.ProjectRootFinderUtil;
+
+@Sources( "file:${projectroot}/.retest/retest.properties" )
```

Comment:
Shouldn't we also read from the system properties? Or is this done with L23?

Reply:
Yes, this is covered by L23: http://owner.aeonbits.org/docs/importing-properties/